lms/static/js/fixtures/discovery.html

```diff
@@ -0,0 +1,52 @@
+<section class="courses-container">
```

Comment:
Looking further in the code, I see that you are using `TemplateHelpers`.  I'm not sure why this fixture is necessary -- do the search views rely on parts of the DOM that they don't render from underscore templates?